examples/embeddable_examples/public/react_embeddables/saved_book/saved_book_editor.tsx

```diff
@@ -64,9 +71,16 @@ export const openSavedBookEditor = (
             attributesManager.numberOfPages.next(initialState.numberOfPages);
             closeOverlay(overlay);
           }}
-          onSubmit={(addToLibrary: boolean) => {
+          onSubmit={async (addToLibrary: boolean) => {
```

Comment:
now that `onSubmit` is aysnc, `SavedBookEditor` should show loading state and disable submit button until `onSubmit` returns to avoid allowing users to click multiple times

Reply:
Good call. Updated this in 8b0809cccd7